LocalyticsXamarin/Android/Properties/AndroidManifest.xml

```diff
@@ -19,22 +19,23 @@
 				<category android:name="com.localytics.xamarin.sample" />
 			</intent-filter>
 		</receiver>-->
-		<service android:name="com.localytics.android.GcmListenerService" android:exported="false">
-			<intent-filter>
-				<action android:name="com.google.android.c2dm.intent.RECEIVE" />
-			</intent-filter>
-		</service>
 		<service android:name="com.localytics.android.InstanceIDListenerService" android:exported="false">
 			<intent-filter>
 				<action android:name="com.google.android.gms.iid.InstanceID" />
 			</intent-filter>
 		</service>
-		<receiver android:name="com.google.android.gms.gcm.GcmReceiver" android:exported="true" android:permission="com.google.android.c2dm.permission.SEND">
-			<intent-filter>
-				<action android:name="com.google.android.c2dm.intent.RECEIVE" />
-				<action android:name="com.google.android.c2dm.intent.REGISTRATION" />
-				<category android:name="com.localytics.android.app" />
-			</intent-filter>
-		</receiver>
+        <receiver
+            android:name="com.google.firebase.iid.FirebaseInstanceIdInternalReceiver"
+            android:exported="false" />
+        <receiver
```

Comment:
Looks to me like you are missing the standard FirebaseMessagingService in here?
Here is the one we use in the SDK for example:
https://github.com/localytics/Android-Client-Library/blob/develop/android-client-library/src/main/AndroidManifest.xml#L41-L48